frontend/src/app/shared/components/project-list/project-list.component.sass

```diff
@@ -31,7 +31,3 @@
     white-space: nowrap
     overflow: hidden
     max-width: 100%
-    color: $spot-color-basic-gray-1
-
-    &_link:hover
-      color: var(--content-link-hover-active-color)
```

Comment:
This results in the global project select showing the projects in the link color. It should be the same as in the project include.
<img width="457" alt="Bildschirmfoto 2022-06-29 um 08 29 27" src="https://user-images.githubusercontent.com/7457313/176368356-7bd8d5fb-30b1-42c5-a56c-73071f65bb3c.png">